db/migrate/20190717025920_create_users.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
+
+      t.timestamps
+
+      add_index :users, :email, unique: true
+      add_column :users, :password_digest, :string
```

Comment:
@shaniajubelag 
You can use the table builder and just add it below email like this
`t.string :password_digest`

![image](https://user-images.githubusercontent.com/20507565/61350983-fa0c0600-a89b-11e9-9bb4-31c75ad088e8.png)